packages/openapi-generator/src/parser/media-type.spec.ts

```diff
@@ -59,6 +73,20 @@ describe('parseMediaType', () => {
     ).toEqual(emptyObjectSchema);
   });
 
+  it('returns parsed media type if there is only application/merge-patch+json', async () => {
+    expect(
+      parseMediaType(
```

Comment:
[q] What happens if `content` contains both  'application/merge-patch+json` and 'application/json' ?